user-guide/Advanced_Modules/Dashboards_app/Overview_Dashboards_UI/Dashboards_app_UI_from_DataMiner_10_0_0_10_0_2_onwards.md

```diff
@@ -33,19 +33,25 @@ The main page of the app consists of a header bar, a sidebar, and a details pane
 
   Right-click in either of the panes or click the ... button to open a menu with the following options:
 
-  - *Delete*: Only available if a dashboard or folder is selected. See [Deleting a dashboard](xref:Deleting_a_dashboard).
-
-  - *Rename*: Only available if a dashboard or a folder is selected.
+  - *Open in new tab*: Opens the selected dashboard in a new browser tab.
 
-  - *Duplicate* or *Copy*: Only available if a dashboard is selected. See [Duplicating a dashboard](xref:Duplicating_a_dashboard)
+  - *Create folder* or *New folder*: Adds a new folder in the selected folder.
 
-  - *Move*: Only available if a dashboard or folder is selected. See [Managing dashboard folders](xref:Managing_dashboard_folders).
+  - *Create dashboard* or *New dashboard*: Adds a new dashboard. By default, it will be added in the selected folder, but a different folder can be specified. See [Creating a completely new dashboard](xref:Creating_a_completely_new_dashboard).
 
   - *Import dashboard*: Allows you to import an example dashboard. By default, it will be added in the selected folder, but a different folder can be specified. See [Creating a dashboard based on an example](xref:Creating_a_dashboard_based_on_an_example).
 
-  - *Create dashboard* or *New dashboard*: Adds a new dashboard. By default, it will be added in the selected folder, but a different folder can be specified. See [Creating a completely new dashboard](xref:Creating_a_completely_new_dashboard).
+  - *Edit*: Available from DataMiner 10.2.0/10.1.12 onwards, if a dashboard folder is selected. Opens a pop-up window where you can rename the folder. 
 
-  - *Create folder* or *New folder*: Adds a new folder in the selected folder.
+    The pop-up window also contains the option *Preserve feed selections*. When this option is selected, any feed selection you make in a dashboard in the folder is preserved when you navigate to another dashboard in the folder. Note that this only applies to the folder itself, not to any other folders it may contain.
```

Comment:
Just a thought, after reading this I felt like it was a bit too repetitive with "in the folder". What do you think of this?

"When this option is enabled, all feed selections are preserved across the dashboards in the same folder. Note that this does not apply to subfolders."

Reply:
I will keep the repetition, since it leaves no room for doubt and may be clearer to someone who is less familiar with the app. Better repetitive than confusing ;-)